docs/HANDOVER.md

```
@@ -0,0 +1,22 @@
+**One tool:** `./scripts/gitcord-handover`
+
+| Who | Command |
+| --- | --- |
+| Old PC | `./scripts/gitcord-handover pack` → one `.tar.gz` on Desktop |
+| New PC | `./scripts/gitcord-handover restore /path/to/that.tar.gz` |
+| Old PC after move | `./scripts/gitcord-handover stop-old` |
+| Anyone | `./scripts/gitcord-handover check` |
+
+**AI:** paste [`HANDOVER_AI_PROMPT.md`](HANDOVER_AI_PROMPT.md) into Cursor and give the `.tar.gz` path.
+
+### Rules
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)